inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -920,6 +920,12 @@ void MKLDNNGraph::SortTopologically() {
 }
 
 void MKLDNNGraph::GetPerfData(std::map<std::string, InferenceEngine::InferenceEngineProfileInfo> &perfMap) const {
+    bool shouldPrintConstantNodes = true;
+
+#ifdef CPU_DEBUG_CAPS
+    shouldPrintConstantNodes = !(getProperty().debugCaps.shouldDumpConstNodes == "NO");
```

Comment:
From my opinion getProperty() is a misname (especially considering what setProperty does)
I know that this is not directly related to your changes, but I propose to replace getProperty() with getConfig() everywhere (getConfig also returns const Config& which seems more properly)